app/src/org/commcare/android/database/connect/models/ConnectJobRecord.java

```diff
@@ -398,6 +411,10 @@ public boolean isMultiPayment() {
         return paymentUnits.size() > 1;
     }
 
+    public boolean getIsUserSuspended(){
```

Comment:
Nit: Let's move the getter and setter so they're next to each other in the code.